pipelines/skylab/smartseq2_single_nucleus/README.md

```diff
@@ -0,0 +1,7 @@
+## Announcing a new site for WARP documentation!
+
+Smart-seq2 Single Sample documentation has moved! Read more about the [Smart-seq2 Single Sample Pipeline](https://broadinstitute.github.io/warp/documentation/Pipelines/Smart-seq2_Single_Sample_Pipeline/) on the new [WARP documentation site](https://broadinstitute.github.io/warp/)!
+
+### Smart-seq2 Single Sample summary
+
+The Smart-seq2 Single Sample pipeline is designed by the [Data Coordination Platform](https://data.humancellatlas.org/about) of the [Human Cell Atlas](https://www.humancellatlas.org/) to process single-cell RNAseq (scRNAseq) data generated by [Smart-seq2 assays](https://www.nature.com/articles/nmeth.2639). The pipeline is designed to process stranded or unstranded, paired- or single-end, scRNA-seq data from an individual cell (to process multiple cells, see the workflow for [Smart-seq2_Multi Sample](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.wdl) pipeline). 
```

Comment:
I know Liz is working on documentation so I'm sure this will get updated, but as it is right now it's not accurate. If this will get merged before the doc updates we may want to remove this file?

Reply:
@ekiernan

Reply:
We left this part so she can fill in the document she has prepared.